configure

```diff
@@ -109,6 +111,7 @@ while true; do
         --valac-flags ) VALACFLAGS="$2"; shift; shift ;;
         --lib-suffix ) LIB_SUFFIX="$2"; shift; shift ;;
         --with-libsoup3 ) USE_SOUP3=yes; shift ;;
+        --no-webrtcdsp ) PLUGIN_RTP_WEBRTC_AUDIO_PROCESSING=no; shift ;;
```

Comment:
It seems this script is trying to be compatible with **GNU Autotools**. I think it's better to use `--without` instead of `--no`.